packages/openneuro-app/src/scripts/users/user-datasets-view.tsx

```diff
@@ -7,47 +11,148 @@ interface User {
 interface Dataset {
   id: string
   created: string
-  ownerId: string
   name: string
-  type: string
+  public: boolean
+  analytics: {
+    views: number
+    downloads: number
+  }
+  stars: [{ userId: string; datasetId: string }]
+  followers: [{ userId: string; datasetId: string }]
+  latestSnapshot?: {
+    id: string
+    size: number
+    issues: [{ severity: string }]
+    created?: string
+  }
 }
```

Comment:
This for example is the same type repeated. I think at least within the context of the user components, these should be consolidated. I'd move these to a user-types.ts file for now and leave any generation from GraphQL and merger with other frontend uses of these types to a separate PR.

Reply:
I made user-types its own file and put it in the types dir.